packages/maskbook/src/components/InjectedComponents/PostDialog.tsx

```diff
@@ -350,6 +396,24 @@ export function PostDialog({ reason: props_reason = 'timeline', ...props }: Post
                         autoPasteFailedRecover: true,
                         relatedText,
                     })
+                } else if (imageEncryptEnabled) {
+                    if (!imgToEncrypt) return
+
+                    const seed = Math.floor(Math.random() * 99999999999999999).toString()
```

Comment:
Do you need crypto level seed? If so, Math.random() is not a good choice

Reply:
i meant to change that